src/components/controller/appLoader.ts

```diff
@@ -3,7 +3,7 @@ import Loader from './loader';
 class AppLoader extends Loader {
     constructor() {
         super('https://newsapi.org/v2/', {
-            apiKey: '1fc3f435fe454787bf34734f14b8ceae', // получите свой ключ https://newsapi.org/
+            apiKey: '2485f009a4ab4482936adb0991cdbf26', // получите свой ключ https://newsapi.org/
```

Comment:
It would be better to define this value as a count variable rather than using magic string

Reply:
Did I do it right? I separated key for 3 parts because of limits for number length.

Reply:
I am confused with definition "a **count** variable". Firstly I thought it  just about making a separate const variable. Then I thought about hexadecimial variable (but it hard to keep and to use it in this way). Now I found this little article about [magic strings](https://dev.to/insomaniacgamer/avoid-magic-strings-and-boost-your-development-36ce) and think that my first conclusion was right. Can you clear up this moment to me?

Reply:
Yes, it was a typo, you are right